department-of-veterans-affairs/abd-vro
# MCP-2664 Slack exam-order endpoint

## What was the problem?
We want an endpoint to trigger a process that slacks all exam orders that have not yet been processed by MAS.

Associated tickets or Slack threads:

- [MCP-2664](https://amida.atlassian.net/browse/MCP-2664)
- Also contains code for [MCP-2731] (https://amida.atlassian.net/browse/MCP-2731)

## How does this fix it?[^1]
Creates an exam-order-slack endpoint which will slack all exam orders that have not been processed by MAS.  aka orderedAt = null.

## How to test this PR
- Run automatedClaim with benefeitClaimId = 1390 and collectionId = 390
- Verify that the audit shows in DB, and that the mock-slack container contains the correct message.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
